### DISCUSSION TITLE
Comparing two profiles

### DISCUSSION THREAD

**Discussion by ajames-branch** [async-profiler/async-profiler#911] [General]
I was wondering if there is any automated way to compare two profiles (flamegraphs) taken by async-profiler for same application?
Something similar to - https://www.brendangregg.com/blog/2014-11-09/differential-flame-graphs.html ?

**Comment by apangin** [comment]
This has been implemented in #442, although the proposed PR got a bit outdated.
Alternatively, you may dump profiles in the collapsed stacks format (`-o collapsed`) and reuse Brendan's script.
